### PR TITLE
feat(plasma-tokens-native): Add version comment in native tokens for all platfroms

### PR DESCRIPTION
**Pull request by neretin-trike**
## Release Notes

Добавлена middleware, которая встраивает в каждый файл с токенами для всех нативных платформ комментарией с версией пакета `plasma-tokens` 

### What/why Changed

Данный функционал нужен для отслеживания изменения в файлах с генерируемыми токенами

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>borderRadius</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_default_react-native--canary.820.6652670506.ts'>borderRadius_default_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_b2c_react-native--canary.820.6652670506.ts'>borderRadius_plasma_b2c_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_web_react-native--canary.820.6652670506.ts'>borderRadius_plasma_web_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberHealth_react-native--canary.820.6652670506.ts'>borderRadius_sberHealth_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_business_react-native--canary.820.6652670506.ts'>borderRadius_sbermarket_business_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_metro_react-native--canary.820.6652670506.ts'>borderRadius_sbermarket_metro_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_react-native--canary.820.6652670506.ts'>borderRadius_sbermarket_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_selgros_react-native--canary.820.6652670506.ts'>borderRadius_sbermarket_selgros_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_wlbusiness_react-native--canary.820.6652670506.ts'>borderRadius_sbermarket_wlbusiness_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberonline_react-native--canary.820.6652670506.ts'>borderRadius_sberonline_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberprime_react-native--canary.820.6652670506.ts'>borderRadius_sberprime_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_stylesSalute_react-native--canary.820.6652670506.ts'>borderRadius_stylesSalute_react-native--canary.820.6652670506.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.820.6652670506.swift'>color_default_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.820.6652670506.kt'>color_default_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.820.6652670506.ts'>color_default_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.820.6652670506.xml'>color_default_xml--canary.820.6652670506.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.820.6652670506.swift'>color_plasma_b2c_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.820.6652670506.kt'>color_plasma_b2c_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.820.6652670506.ts'>color_plasma_b2c_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.820.6652670506.xml'>color_plasma_b2c_xml--canary.820.6652670506.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.820.6652670506.swift'>color_plasma_web_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.820.6652670506.kt'>color_plasma_web_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.820.6652670506.ts'>color_plasma_web_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.820.6652670506.xml'>color_plasma_web_xml--canary.820.6652670506.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.820.6652670506.swift'>color_sberHealth_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.820.6652670506.kt'>color_sberHealth_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.820.6652670506.ts'>color_sberHealth_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.820.6652670506.xml'>color_sberHealth_xml--canary.820.6652670506.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.820.6652670506.swift'>color_sbermarket_business_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.820.6652670506.kt'>color_sbermarket_business_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.820.6652670506.ts'>color_sbermarket_business_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.820.6652670506.xml'>color_sbermarket_business_xml--canary.820.6652670506.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.820.6652670506.swift'>color_sbermarket_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.820.6652670506.kt'>color_sbermarket_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.820.6652670506.swift'>color_sbermarket_metro_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.820.6652670506.kt'>color_sbermarket_metro_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.820.6652670506.ts'>color_sbermarket_metro_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.820.6652670506.xml'>color_sbermarket_metro_xml--canary.820.6652670506.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.820.6652670506.ts'>color_sbermarket_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.820.6652670506.swift'>color_sbermarket_selgros_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.820.6652670506.kt'>color_sbermarket_selgros_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.820.6652670506.ts'>color_sbermarket_selgros_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.820.6652670506.xml'>color_sbermarket_selgros_xml--canary.820.6652670506.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.820.6652670506.swift'>color_sbermarket_wlbusiness_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.820.6652670506.kt'>color_sbermarket_wlbusiness_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.820.6652670506.ts'>color_sbermarket_wlbusiness_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.820.6652670506.xml'>color_sbermarket_wlbusiness_xml--canary.820.6652670506.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.820.6652670506.xml'>color_sbermarket_xml--canary.820.6652670506.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.820.6652670506.swift'>color_sberonline_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.820.6652670506.kt'>color_sberonline_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.820.6652670506.ts'>color_sberonline_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.820.6652670506.xml'>color_sberonline_xml--canary.820.6652670506.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.820.6652670506.swift'>color_sberprime_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.820.6652670506.kt'>color_sberprime_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.820.6652670506.ts'>color_sberprime_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.820.6652670506.xml'>color_sberprime_xml--canary.820.6652670506.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.820.6652670506.swift'>color_stylesSalute_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.820.6652670506.kt'>color_stylesSalute_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.820.6652670506.ts'>color_stylesSalute_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.820.6652670506.xml'>color_stylesSalute_xml--canary.820.6652670506.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.820.6652670506.ts'>shadow_default_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.820.6652670506.ts'>shadow_plasma_b2c_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.820.6652670506.ts'>shadow_plasma_web_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.820.6652670506.ts'>shadow_sberHealth_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.820.6652670506.ts'>shadow_sbermarket_business_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.820.6652670506.ts'>shadow_sbermarket_metro_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.820.6652670506.ts'>shadow_sbermarket_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.820.6652670506.ts'>shadow_sbermarket_selgros_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.820.6652670506.ts'>shadow_sbermarket_wlbusiness_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.820.6652670506.ts'>shadow_sberonline_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.820.6652670506.ts'>shadow_sberprime_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.820.6652670506.ts'>shadow_stylesSalute_react-native--canary.820.6652670506.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>spacing</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_default_react-native--canary.820.6652670506.ts'>spacing_default_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_b2c_react-native--canary.820.6652670506.ts'>spacing_plasma_b2c_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_web_react-native--canary.820.6652670506.ts'>spacing_plasma_web_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberHealth_react-native--canary.820.6652670506.ts'>spacing_sberHealth_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_business_react-native--canary.820.6652670506.ts'>spacing_sbermarket_business_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_metro_react-native--canary.820.6652670506.ts'>spacing_sbermarket_metro_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_react-native--canary.820.6652670506.ts'>spacing_sbermarket_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_selgros_react-native--canary.820.6652670506.ts'>spacing_sbermarket_selgros_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_wlbusiness_react-native--canary.820.6652670506.ts'>spacing_sbermarket_wlbusiness_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberonline_react-native--canary.820.6652670506.ts'>spacing_sberonline_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberprime_react-native--canary.820.6652670506.ts'>spacing_sberprime_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_stylesSalute_react-native--canary.820.6652670506.ts'>spacing_stylesSalute_react-native--canary.820.6652670506.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.820.6652670506.swift'>typo_mage_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.820.6652670506.kt'>typo_mage_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.820.6652670506.ts'>typo_mage_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.820.6652670506.swift'>typo_plasma_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.820.6652670506.kt'>typo_plasma_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.820.6652670506.ts'>typo_plasma_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.820.6652670506.swift'>typo_ruler_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.820.6652670506.kt'>typo_ruler_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.820.6652670506.ts'>typo_ruler_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.820.6652670506.swift'>typo_sage_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.820.6652670506.kt'>typo_sage_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.820.6652670506.ts'>typo_sage_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.820.6652670506.swift'>typo_sbermarket_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.820.6652670506.kt'>typo_sbermarket_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.820.6652670506.ts'>typo_sbermarket_react-native--canary.820.6652670506.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.820.6652670506.swift'>typo_soulmate_ios-swift--canary.820.6652670506.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.820.6652670506.kt'>typo_soulmate_kotlin--canary.820.6652670506.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.820.6652670506.ts'>typo_soulmate_react-native--canary.820.6652670506.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-820/

File: utils/plasma-tokens-native/src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export const comments = (content: string) => ({
+    arrows: `<!-- ${content} -->`,
+    slashes: `// ${content}`,
+});
+
+export type CommentsType = keyof ReturnType<typeof comments>;
+
+const getPackageVersion = () => {
+    const packages = path.join(__dirname, '../data/package.json');
+    const data = JSON.parse(fs.readFileSync(packages, 'utf-8'));
```
Comment (Yeti-or) [inline utils/plasma-tokens-native/src/utils.ts:15]: а мы `fs-extra` не хотим использовать тут ?

File: utils/plasma-tokens-native/package.json [hunk 2]
```diff
@@ -1,44 +1,37 @@
...
-    "ts-node": "^10.9.1"
-  },
-  "author": "Salute Frontend Team <salute.developers@gmail.com>",
-  "license": "MIT",
-  "publishConfig": {
-    "access": "public"
-  },
-  "bugs": {
-    "url": "https://github.com/salute-developers/plasma/issues"
-  }
```
Comment (Yeti-or) [inline utils/plasma-tokens-native/package.json:43]: как меня задолбали постоянные изменения в package.json

**Review by Yeti-or — APPROVED** [review]

File: utils/plasma-tokens-native/src/utils.ts (hunk 1 above)
Comment (neretin-trike) [reply utils/plasma-tokens-native/src/utils.ts:15]: мы много раз обсуждали, но в итоге всегда договаривались, что обычного `fs` достаточно

**Review by Yakutoc — APPROVED** [review]

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-tokens-native@1.14.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.240.0, @salutejs/plasma-core@1.137.0, @salutejs/plasma-hope@1.238.0, @salutejs/plasma-icons@1.168.0, @salutejs/plasma-new-hope@0.9.0, @salutejs/plasma-sb-utils@0.135.0, @salutejs/plasma-temple-docs@0.177.0, @salutejs/plasma-temple@1.186.0, @salutejs/plasma-tokens-native@1.14.0, @salutejs/plasma-ui-docs@0.226.0, @salutejs/plasma-ui@1.218.0, @salutejs/plasma-web-docs@0.194.0, @salutejs/plasma-web@1.240.0, @salutejs/plasma-website@0.202.0 🚀

**Pull request merged**